src/utils/url-utils.js

```diff
@@ -49,15 +49,24 @@ function ensureIsFullyQualifiedUrl(pathOrUrl, fallbackBase) {
 
 function getImageUrl({ cdnRootUrl, sourceType, sourceUrl }) {
   switch (sourceType) {
-    case IMAGE_SOURCE_TYPE.external:
-      return sourceUrl || null;
     case IMAGE_SOURCE_TYPE.internal:
       return sourceUrl ? `${cdnRootUrl}/${sourceUrl}` : null;
+    case IMAGE_SOURCE_TYPE.external:
```

Comment:
Actually, we could make `return sourceUrl || null;` the default case (like in the function below). We only need to prepend in case of internal URLs for both images and video/audio.

Reply:
Agreed, simplified 👍